### ISSUE TITLE
How can I get the dataset?

### ISSUE THREAD

**Issue by nanamaru0203** [sukjulian/segnn-hemodynamics#1]
I cannot access the url you share [here](https://drive.google.com/file/d/10p_O9R0op7zLwo1w6qGv8FjK3S-EXBNn/view?usp=sharing).
Does any accident happen?
Also, can I use full dataset (not tiny one)?

**Comment by sukjulian** [comment]
I moved the dataset to a different server and updated the download link. Could you please try if everything works?

**Comment by sukjulian** [comment]
Hi Kanta,

 

 

I am currently looking into ways of making the full-size dataset available since it is quite large (40 GB). May I ask for what are you planning to use it?

 

 

Best regards,

 

Julian

 
 

Gesendet: Freitag, 19. Januar 2024 um 10:45 Uhr
Von: "Kanta Shimizu" ***@***.***>
An: "sukjulian/segnn-hemodynamics" ***@***.***>
Cc: "Subscribed" ***@***.***>
Betreff: [sukjulian/segnn-hemodynamics] How can I get the dataset? (Issue #1)


 

I cannot access the url you share here.
Does any accident happen?
Also, can I use full dataset (not tiny one)?

**Comment by nanamaru0203** [comment]
Thank you for your quick response.
I was able to download the data set without any problems.

I am currently working on a project on CFD surrogate model for cerebral aneurysms.

However, the problem is that the environment I have does not allow me to quickly prepare a sufficient dataset.

So, as a first step of our research, I would like to make a comparison with your methodology using your dataset, which has a high affinity even if it is not about cerebral aneurysms.

In other words, I would like to facilitate our project by training and testing our network using your full dataset.

We would be grateful if you would positively consider releasing the full dataset.
